client/jetpack-cloud/components/user-feedback-modal-form/index.tsx

```diff
@@ -37,9 +40,10 @@ export default function UserFeedbackModalForm( { show, onClose }: Props ) {
 		if ( ! hasCompletedForm ) {
 			return;
 		}
-
-		// TODO: send feedback to backend
-	}, [ hasCompletedForm ] );
+		// Remove the hash from the URL.
+		const sourceUrl = window.location.href.split( '#' )[ 0 ];
```

Comment:
I suggest combining `window.location.origin` and `window.location.pathname` to strip down any query strings after **?**. wdyt?

Reply:
Makes sense. I have now updated it.